.github/goreleaser.yaml

```diff
@@ -487,10 +591,19 @@ release:
   draft: true
   replace_existing_draft: true
   prerelease: auto
+  # make_latest: {{ if .IsNightly }}true{{ else }}false{{ end }}
+  # make_latest: {{ if eq .Env.TAG_VERSION "master" }}false{{ else }}true{{ end }}
```

Comment:
leftover?

Reply:
This was actually left on purpose to get an opinion...
Unfortunately templating this variable is not possible, this is the only difference between `master` and `nightly` build. I solved that by skipping `publish` for master, but I don't know if this is the right path